product/charts/layouts/daily_perf_charts/MiddlewareDatasource.yaml

```diff
@@ -0,0 +1,14 @@
+# Middleware Datasource daily performance chart layouts
+---
+- :title: Availability (Connections)
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1662329/16008795/eeee810e-3179-11e6-8cbe-9f6e8cb91266.png)
Goal was to reproduce the same graphs supported on the original hawkular UI.
Titles and descriptions were discussed from UX team, I'm just porting them into MiQ.